src/main/java/seedu/address/model/event/Time.java

```diff
@@ -17,9 +17,14 @@
  */
 public class Time {
     public static final String MESSAGE_CONSTRAINTS = "Date time format not accepted, the following are accepted:\n"
-            + "dd-MM-yyyy HH:mm";
+            + "dd-MM-yyyy HH:mm\n"
+            + "yyyy-dd-MM HH:mm\n"
+            + "[.][/][-][:] can be used interchangeably to indicate date, time, year of the event.";
 
-    private static final String STANDARD_TIME_PATTERN = "dd-MM-yyyy HH:mm";
+    private static final String symbol = "[.][/][-][:]";
+    private static final String STANDARD_TIME_PATTERN =
+            "[dd" + symbol + "MM" + symbol + "yyyy HH" + symbol + "mm]"
+                    + "[yyyy" + symbol + "dd" + symbol + "MM HH" + symbol + "mm]";
```

Comment:
i think this straight up breaks the parser as it relies on this field to check if the input format is correct. i think this branch should only be merged when we actually change `isValidTime` and `parse` below.

Reply:
Why do isValidTime and parse need to be changed?

Reply:
sorry, this is a misinterpretation on my part.